requirements_test_all.txt

```diff
@@ -596,6 +596,9 @@ nuheat==0.3.0
 # homeassistant.components.numato
 numato-gpio==0.10.0
 
+# homeassistant.components.spokestack_tts
+numpy==1.19.2
```

Comment:
you cannot use different numpy than the rest of HA.

Reply:
Thank you for the catch. I removed the other references and just missed this.